script/script.js

```diff
@@ -1,58 +1,62 @@
-$('.idea-input').keyup(enabledBtn);
-$('.main-title').keyup(enabledBtn);
-
-$(document).ready(function() {
+$(document).ready(onLoad);
 // getStoredCards()
-getIdea();
-cardRestore();
-
-});
-
-
-function getIdea(id) {
-	
-	var retrieveIdea = JSON.parse(localStorage.getItem(id));
 
-	if (retrieveIdea ) {
-		return retrieveIdea;
-	} else {
-		return [];
-	}
+//TL: made on page-load function non-anonymous.
+function onLoad(){
+	cardRestore();
+	$('#title-input').focus();
 }
 
-function enabledBtn() {
-    if ( $('.idea-input').val() === "" && $('.main-title').val() === "") {
-      $('#submit-button').attr("disabled", true)
-    } else {
-      $('#submit-button').attr("disabled", false);
-    }
-}
-
-$('#submit-button').on('click', addIdea);
-
 function cardRestore(){
 	var keys = Object.keys(localStorage)
 	keys.forEach(function(key){
 		localStorage[key]
-		console.log(localStorage[key])
 		prependIdea(JSON.parse(localStorage[key]))
-	})
+})
+}
+
+// getIdea(); I think this function below is not doing anything (as you said Friday?)
+// function getIdea(id) { * this function *
+// 	var retrieveIdea = JSON.parse(localStorage.getItem(id));
+// 	if (retrieveIdea) {
+// 		return retrieveIdea;
+// 	} else {
+// 		return [];
+// 	}
+// }
+
+$('.idea-input').keyup(enabledBtn);
+$('.main-title').keyup(enabledBtn);
+
+function enabledBtn() {
+    if ( $('.idea-input').val() === "" || $('.main-title').val() === "") {
+      $('.enterButton').prop('disabled', true);
+    } else {
+      $('#submit-button').removeAttr('disabled');
```

Comment:
Just a question.  Is there a reason to choose .removeAttr over .prop('disabled', false) or .attr('disabled', false) or .toggle?